graph/pull.go

```diff
@@ -516,15 +524,19 @@ func (s *TagStore) pullV2Tag(r *registry.Session, out io.Writer, endpoint *regis
 
 				if !verifier.Verified() {
 					logrus.Infof("Image verification failed: checksum mismatch for %q", di.digest.String())
+					out.Write(sf.FormatProgress(stringid.TruncateID(img.ID), "Download complete, Checksum failed", nil))
+					di.validChecksum = false
```

Comment:
isn't it will be simpler to replace `verified` with `di.validChecksum` everywhere below?